app/controllers/serviceLearning/routes.py

```diff
@@ -95,13 +101,27 @@ def slcCreateCourse():
 @serviceLearning_bp.route('/serviceLearning/saveProposal', methods=['POST'])
 def slcSaveContinue():
     """Will update the the course proposal and return an empty string since ajax request needs a response
-    Also, it updates the course status as 'Incomplete'"""
+    Also, it updates the course status as 'in progress'"""
     course = updateCourse(request.form.copy())
-    course.status = CourseStatus.INCOMPLETE
-    course.save()
+    if not course:
+        flash("Error saving changes", "danger")
+    else:
+        course.status = CourseStatus.IN_PROGRESS
+        course.save()
+    return ""
 
+@serviceLearning_bp.route('/serviceLearning/saveExit', methods=['POST'])
+def slcSaveExit():
+    """Will update the the course proposal and return an empty string since ajax request needs a response
+    Also, it updates the course status as 'in progress'"""
+    course = updateCourse(request.form.copy())
```

Comment:
This (and above) assumes that `updateCourse` succeeds. Either update course needs to not catch the blanket exception, or this code needs to handle a tuple return value of (False, e), which is what `updateCourse` returns if it fails.